app/views/posts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Create a New Post</h1>
+
+<%= form_with(model: @post, url: create_post_path(user_id: params[:user_id]), local: true) do |form| %>
+  <%= form.label :title %>
+  <%= form.text_field :title %>
+
+  <%= form.label :text %>
+  <%= form.text_area :text %>
+
+  <%= form.submit "Create Post" %>
+<% end %>
```

Comment:
- [ ] Great job implementing forms for your app. However, there's a glitch that needs to be fixed. I was able to create posts from the terminal with the user_id. However, when I tried to create a post from the form, I got below error message. Kindly look into it and fix.

### Post Creating Error


![post-err](https://github.com/Jeddaa/rails-blog/assets/26597432/3ad8cd64-6a9d-4027-8f64-2c6b835dd169)


[post-error.webm](https://github.com/Jeddaa/rails-blog/assets/26597432/def27543-6454-444f-9db6-c62521cf2464)

Reply:
hello @Qoosim did you manually set the primary key when you created posts from the terminal with the user_id? because that might be the source of the error as the primary key is set to autogenerate by rails

Reply:
I am able to create a post
<img width="1440" alt="Screenshot 2023-07-21 at 17 11 31" src="https://github.com/Jeddaa/rails-blog/assets/44745855/1e921e63-0034-4dc2-9c88-e6543d55d600">

Reply:
@DeliceLydia yes i am able to create post too